server/game/cards/events/03/evenhandedjustice.js

```diff
@@ -0,0 +1,65 @@
+const _ = require('underscore');
+
+const DrawCard = require('../../../drawcard.js');
+
+class EvenHandedJustice extends DrawCard {
+
+    canPlay(player, card) {
+        if(player !== this.controller || this !== card) {
+            return false;
+        }
+
+        return (_.every(this.game.getPlayers(), this.hasStandingCharacters)
+                && super.canPlay());
+    }
+
+    hasStandingCharacters(player) {
+        return (player.cardsInPlay
+                .filter(card => card.getType() === 'character' && !card.kneeled)
+                .length > 0);
+    }
+
+    play(player) {
+        var waitingPrompt = 'Waiting for opponent to use ' + this.name;
+
+        this.toKneel = [];  // characters to kneel (later)
+
+        // kneel opponent character
+        this.game.promptForSelect(player, {
+            activePromptTitle: 'Select a standing character controlled by your opponent',
+            waitingPromptTitle: waitingPrompt,
+            cardCondition: card =>
+                !card.kneeled
+                && card.getType() === 'character'
+                && card.controller !== this.controller,
+            onSelect: (player, card) => this.toKneel.push(card),
+        });
+
+        // kneel controller character
+        this.game.promptForSelect(player, {
+            activePromptTitle: 'Select a standing character of yours',
+            waitingPromptTitle: waitingPrompt,
+            cardCondition: card =>
+                !card.kneeled
+                && card.getType() === 'character'
+                && card.controller === this.controller,
+            onSelect: (player, card) => this.onCardsSelected(player, card),
+        });
+    }
+
+    onCardsSelected(player, card) {
+        this.toKneel.push(card);
+        _.each(this.toKneel, card => card.controller.kneelCard(card));
+
+        var kneeledCards = _.map(this.toKneel, card => card.name);
+        this.game.addMessage('{0} uses {1} to kneel {2}',
```

Comment:
Gah, I'd rather not copy paste that in another card. If you don't mind accepting it as is (i.e., with just non-hoverable card names in the message), I prefer to wait for that "markup" feature to be available as a helper function from somewhere. If you really want this as a requirement to land the implementation of Even-Handed Justice, I guess I could do the helper refactoring myself, but no promises ;-)